CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,27 @@
+#Code of Conduct
+
+## Summary
+
+Harassment in code and discussion or violation of physical boundaries is completely unacceptable anywhere in Opal’s project codebases, issue trackers, chatrooms, mailing lists, meetups, etc. Violations will result in being warned, blocked or banned by the core team at or before the 3rd violation.
+
+## In detail
+
+Harassment includes offensive verbal comments related to gender identity, gender expression, sexual orientation, disability, physical appearance, body size, race, religion, sexual images, deliberate intimidation, stalking, sustained disruption, and unwelcome sexual attention.
```

Comment:
@bright1eyes I recommend you bring these suggestions upstream to the [Contributor Covenant](https://github.com/CoralineAda/contributor_covenant/issues).